src/lib/actions/analytics.ts

```diff
@@ -71,14 +71,10 @@ export const trackEvent = async (name: string, data: object = {}) => {
 
 export function isTrackingAllowed() {
     if (ENV.TEST) {
-        return;
+        return false;
     }
     if (window.navigator?.doNotTrack) {
```

Comment:
`navigator.doNotTrack` is deprecated https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack

Reply:
Addressed.